src/libNode/Node.cpp

```diff
@@ -153,8 +153,9 @@ bool Node::DownloadPersistenceFromS3() {
   LOG_MARKER();
   string output;
   // TBD - find better way to capture the exit status of command
-  SysCommand::ExecuteCmdWithOutput("./downloadIncrDB.py " + STORAGE_PATH + "/",
-                                   output);
+  string excludembtxns = LOOKUP_NODE_MODE ? "false" : "true";
+  SysCommand::ExecuteCmdWithOutput(
+      "./downloadIncrDB.py " + STORAGE_PATH + "/ " + excludembtxns, output);
```

Comment:
Maybe can directly write the condition inside, to avoid the extra variable declaration?

Reply:
I would prefer clean readability over extra variable in this case. Can change if you insist?